msal/src/Microsoft.Identity.Client/ClientApplicationBase.cs

```diff
@@ -52,12 +53,26 @@ public abstract partial class ClientApplicationBase
         protected const string DefaultAuthority = "https://login.microsoftonline.com/common/";
 
         /// <summary>
-        /// 
+        /// Constructor of the base application
         /// </summary>
-        /// <param name="clientId"></param>
-        /// <param name="authority"></param>
-        /// <param name="redirectUri"></param>
-        /// <param name="validateAuthority"></param>
+        /// <param name="clientId">Client ID (also named Application ID) of the application as registered in the 
+        /// application registration portal (https://aka.ms/msal-net-register-app)</param>
+        /// <param name="authority">Authority of the Security service token (STS) from which MSAL.NET will acquire the tokens.
+        /// 
+        /// Usual authorities are:
```

Comment:
>Usual [](start = 12, length = 5)

not sure this is the right word. 

Authority endpoints perhaps?

Reply:
changed